x-pack/plugins/reporting/server/lib/store/store.ts

```diff
@@ -142,6 +200,7 @@ export class ReportingStore {
         index: report._index,
         if_seq_no: report._seq_no,
         if_primary_term: report._primary_term,
+        refresh: true,
```

Comment:
These DSL blocks originally came from ESQueue code, but I think this `refresh` should have been added long ago.